docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -2,6 +2,12 @@
 
 If your question is not covered by the topics below, use the [OpenVINO Support page](https://software.intel.com/en-us/openvino-toolkit/documentation/get-started), where you can participate on a free forum.
 
+@sphinxdirective
+
+.. _mo_question_1:
+
+@endsphinxdirective
```

Comment:
can you please clarify for what these changes are? Is it only for the first question?